cchecker_web/api.py

```diff
@@ -156,11 +156,11 @@ def prettify(ugly):
 
     :param str ugly: An ugly string
     '''
-    pretty = ugly.replace('-', ' ')
+    pretty = ugly.replace('-', ' ').replace('_', ' ')
```

Comment:
Don't call replace twice.

~~Use something like `ugly.translate(str.maketrans('-_', '   '))`, which doesn't need regex or to iterate the string twice.~~

Edit: not in Python2

Reply:
That should be two spaces inside the quotes there, but for some reason, the markdown parser is eating them.

Reply:
OK